cmu_pbspro_connector/bin/cmu_pbs_server_scheduler_restart

```diff
@@ -48,9 +50,17 @@ node=`hostname`
 # If node is _not_ the PBS_SERVER then echo simple error message
 if [ "${node}" == "${PBS_SERVER}" ]; then
     case `basename $0` in
-        *stop)    /etc/init.d/pbs stop ;;
-        *start)   /etc/init.d/pbs start ;;
+        *stop)       /etc/init.d/pbs stop ;;
+        *restart)    /etc/init.d/pbs restart ;;
+        *start)      /etc/init.d/pbs start ;;
     esac
 else
-    echo "Node: $node,  Error: Not the PBS_SERVER."
+    case `basename $0` in
```

Comment:
this script is about restarting server/scheduler and not other nodes. basic behaviour is changed by adding other nodes to restart.